### PR TITLE
[16.0][MIG] partner_contact_sale_info_propagation: Migration to 16.0

### PR DESCRIPTION
**Pull request by KM-joel**
Migration to version 16.0

 - **partner_contact_sale_info_propagation**
 
The field team_id invisible=1 is already available on the base, so I remove the inheritance from the res.partner view and all the functionalities are okey and I changed the version in the manifest

### CODE REVIEW

File: partner_contact_sale_info_propagation/tests/test_partner_contact_sale_info_propagation.py [hunk 1]
```diff
@@ -54,9 +54,6 @@ def test_onchange_parent_id_with_values_false(self):
         partner_child = self.partner_model.create({"name": "Parent child"})
         partner_child.write({"parent_id": self.parent_company.id})
         onchange_result = partner_child.onchange_parent_id()
-        self.assertEqual(
```
Comment (pedrobaeza) [inline partner_contact_sale_info_propagation/tests/test_partner_contact_sale_info_propagation.py]: If you have to remove this, it means something is failing. Please also use `Form` for not having to manually call onchanges.

Comment (KM-joel) [reply partner_contact_sale_info_propagation/tests/test_partner_contact_sale_info_propagation.py]: Hey @pedrobaeza 

I used the Form as you advised me and changed the fields_view_get() method to get_view().

**Comment by pedrobaeza** [comment]
Please squash together your migration commits:

https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-your-own-commits

**Comment by KM-joel** [comment]
Hey @pedrobaeza,

it's done, I have squash together my migration commits.

**Review by pedrobaeza — APPROVED** [review]

**Comment by KM-joel** [comment]
Hey @ernestotejeda @victoralmau,

Request review.

**Review by victoralmau — APPROVED** [review]
Code review OK

**Comment by pedrobaeza** [comment]
Well, you have squashed now all together the pre-commit stuff. I meant the migration commits (there were several), not the pre-commit one.

**Comment by KM-joel** [comment]
Hey @pedrobaeza ,

> Well, you have squashed now all together the pre-commit stuff. I meant the migration commits (there were several), not the pre-commit one.

Is there a possibility to redo the pre-commit (pre-commit stuff.) ?

**Comment by pedrobaeza** [comment]
I think the best you can do is:

- Copy the module folder in a separate place.
- Do `git reset --hard HEAD~1` for going back to the previous state before doing anything.
- Run pre-commit + commit pre-commit stuff.
- Remove the module folder and copy back the backup you made.
- Commit latest changes in the migration commit.
- Do `git push -f` with the result.

**Comment by KM-joel** [comment]
> I think the best you can do is:
> 
>     * Copy the module folder in a separate place.
> 
>     * Do `git reset --hard HEAD~1` for going back to the previous state before doing anything.
> 
>     * Run pre-commit + commit pre-commit stuff.
> 
>     * Remove the module folder and copy back the backup you made.
> 
>     * Commit latest changes in the migration commit.
> 
>     * Do `git push -f` with the result.

@pedrobaeza 

it is done.

**Comment by pedrobaeza** [comment]
Great

/ocabot merge nobump

**Pull request merged**